README.md

```diff
@@ -14,42 +14,82 @@ We use this prototype internally to LeanIX to generate SBOMs for our projects. I
 
 1. Run the Docker container from the latest image:
 
+GitHub:
 ```console
-docker run --pull=always \
+docker run --pull=always --rm \
            -v /var/run/docker.sock:/var/run/docker.sock \
            -v <temp-folder-to-be-used-for-storing-data>:/tempDir \
            -e MOUNTED_VOLUME='<temp-folder-to-be-used-for-storing-data>' \
+           -e LEANIX_HOST='<leanix-workspace-host>' \
+           -e LEANIX_REGION='<leanix-workspace-region>' \
            -e LEANIX_TOKEN='<leanix-technical-user-token>' \
+           -e GIT_PROVIDER='GITHUB' \
            -e GITHUB_TOKEN='<github-token>' \
            -e GITHUB_ORGANIZATION='<github-organization>' \
-           leanixacrpublic.azurecr.io/vsm-sbom-booster -rm
+           leanixacrpublic.azurecr.io/vsm-sbom-booster
 ```
+
+GitLab:
+```console
+docker run --pull=always --rm \
+           -v /var/run/docker.sock:/var/run/docker.sock \
+           -v <temp-folder-to-be-used-for-storing-data>:/tempDir \
+           -e MOUNTED_VOLUME='<temp-folder-to-be-used-for-storing-data>' \
+           -e LEANIX_HOST='<leanix-workspace-host>' \
+           -e LEANIX_REGION='<leanix-workspace-region>' \
+           -e LEANIX_TOKEN='<leanix-technical-user-token>' \
+           -e GIT_PROVIDER='GITLAB' \
+           -e GITLAB_TOKEN='<github-token>' \
```

Comment:
minor: please enter the <gitlab-token> as placeholder for the `GITLAB_TOKEN`

Reply:
I am missing something in your comment. Please enter what here?

Reply:
sorry. you currently ahve GITLAB_TOKEN = '<github-token>' prob it should be '<gitlab-token>', right?

Reply:
I think you are saying the value inside the quotes still refers to GH even though it is the GL command. You are correct I overlooked that value, and corrected it in my most recent commit. If that isn't what you mean, then maybe open a new comment on the new commit.